packages/venia-concept/.graphqlconfig

```diff
@@ -8,7 +8,7 @@
                 },
                 "validate-magento-pwa-queries": {
                     "clients": ["apollo", "literal"],
-                    "filesGlob": "src/**/*.{js,graphql,gql}"
+                    "filesGlob": "../{venia-ui,venia-concept}/{lib,src}/**/**/*.{js,graphql,gql}"
```

Comment:
I'm bad with patterns...do I need the extra `/**`?

Reply:
I don't think you do.